### DISCUSSION TITLE
Are there any plans for a 1.19.3 version?

### DISCUSSION THREAD

**Discussion by DBPT210** [stashymane/extra-sounds#98] [Q&A]
Hello. This is probably a very frequently asked question, but are there any plans for a 1.19.3 version?

**Comment by BlueSky-fur** [comment]
No intent to hijack your discussion but its better than to create a new one.

Is this project still maintained and is there a goal for a 1.19.4 version or is this project EoL?

_(I am not asking for an ETA, just general curiosity if it will keep getting updates!)_
